packages/eslint-mdx/src/parser.ts

```diff
@@ -191,7 +193,9 @@ export class Parser {
 
           let normalized = this.normalizeJsxNode(node, parent, options)
           normalized = Array.isArray(normalized) ? normalized : [normalized]
-          normalized.forEach(_node => this._nodeToAst(_node, options))
+          for (const _node of normalized) {
```

Comment:
I'm curious why `_` prefix is used here but not else where.
Usually it denotes an attribute which is publicly visible, but should be treated as a private member.
But this appears to be already block scoped :thinking:

Reply:
There is a top shadow `node`, maybe I should rename it to `normalizedNode`?

Reply:
> `normalizedNode`

that could be a bit clearer